internal/distro/fedora/distro.go

```diff
@@ -145,12 +145,13 @@ var (
 			Locale:          common.ToPtr("en_US.UTF-8"),
 			EnabledServices: iotServices,
 		},
-		rpmOstree:        true,
-		bootISO:          true,
-		image:            iotInstallerImage,
-		buildPipelines:   []string{"build"},
-		payloadPipelines: []string{"anaconda-tree", "rootfs-image", "efiboot-tree", "bootiso-tree", "bootiso"},
-		exports:          []string{"bootiso"},
+		rpmOstree:           true,
+		bootISO:             true,
+		image:               iotInstallerImage,
+		buildPipelines:      []string{"build"},
+		payloadPipelines:    []string{"anaconda-tree", "rootfs-image", "efiboot-tree", "bootiso-tree", "bootiso"},
+		exports:             []string{"bootiso"},
+		basePartitionTables: iotBasePartitionTables,
```

Comment:
this isn't needed @7flying (as this is the anaconda installer)

Reply:
I haven't had time to check yet but that might set up ext4 by default on the anaconda installer, which was something that we also needed

Reply:
I confirm that it doesn't change it to `ext4`, will delete update the commit in a second
```
Filesystem     Type      Size  Used Avail Use% Mounted on
devtmpfs       devtmpfs  4.0M     0  4.0M   0% /dev
tmpfs          tmpfs     1.9G     0  1.9G   0% /dev/shm
tmpfs          tmpfs     777M  996K  776M   1% /run
/dev/vda3      btrfs      14G  1.5G   12G  12% /sysroot
tmpfs          tmpfs     1.9G     0  1.9G   0% /tmp
/dev/vda3      btrfs      14G  1.5G   12G  12% /var/home
/dev/vda2      ext4      974M  100M  807M  11% /boot
/dev/vda1      vfat      599M  7.1M  592M   2% /boot/efi
tmpfs          tmpfs     389M     0  389M   0% /run/user/1000
```